packages/components/file-input/src/FileInput.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ChangeEvent } from 'react';
+import { Button } from '@gpn-prototypes/vega-button';
+import { block } from 'bem-cn';
+
+import './FileInput.css';
+
+type ButtonProps = React.ComponentProps<typeof Button>;
+
+export type FileInputProps = JSX.IntrinsicElements['input'] & {
```

Comment:
я тут вспомнила, разве не хотели так называть тип пропсов?

```suggestion
export type FileInputAPI = JSX.IntrinsicElements['input'] & {
```

Reply:
Это для хуков было